app/javascript/modCenter/singleArticle/util.js

```diff
@@ -1,20 +1,23 @@
-const time_options = {
+const timeOptions = {
   hour: 'numeric',
   minute: 'numeric',
 };
 
-const date_options = {
+const dateOptions = {
   month: 'short',
   day: 'numeric',
 };
 
-export const formatDate = (timestamp, currentLocale) => {
+const currentLocale = window.navigator.languages
+  ? window.navigator.languages[0]
+  : window.navigator.userLanguage || window.navigator.language;
```

Comment:
You can use [optional chaining](const currentLocale = window.navigator.languages?.[0] || window.navigator.userLanguage || window.navigator.language;
) to make this a oneliner
```suggestion
const currentLocale = window.navigator.languages?.[0] || window.navigator.userLanguage || window.navigator.language;

```

Reply:
Perfect! Thank you ❤️